packages/core/src/textures/resources/Resource.ts

```diff
@@ -194,17 +211,22 @@ export class Resource
      * @param {PIXI.GLTexture} glTexture - texture instance for this webgl context
      * @returns {boolean} `true` is success
      */
-    style(renderer, baseTexture, glTexture) // eslint-disable-line no-unused-vars
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    /* eslint-disable @typescript-eslint/ban-ts-ignore */
+    // @ts-ignore
+    style(renderer: Renderer, baseTexture: BaseTexture, glTexture: GLTexture): boolean
```

Comment:
Can use _ here

Reply:
In this particular case, user can override it in children and IDE will automatically add `_` for all three fields there. I'm not sure that its acceptable.